apiclient/harvester_api/managers.py

```diff
@@ -730,3 +730,11 @@ def softreboot(self, name, namespace=DEFAULT_NAMESPACE, *, raw=False):
         path = self.PATH_fmt.format(uid=f"/{name}", ns=namespace)
         params = dict(action="softreboot")
         return self._create(path, raw=raw, params=params)
+
+
+class VirtualMachineInstanceManager(BaseManager):
+    PATH_fmt = "v1/harvester/kubevirt.io.virtualmachineinstances/{ns}{uid}"
+
+    def get(self, name="", namespace=DEFAULT_NAMESPACE, *, raw=False, **kwargs):
```

Comment:
this method has been implemented as `api.vms.get_status`, consider that we will not let user to manipulate VMI directly, so we will not add VMI Manager.